merge.go

```diff
@@ -112,7 +112,7 @@ func duplicateCreateNode(delta []Delta, msg string) bool {
 	return false
 }
 
-func Merge(memGraph *lpg.Graph, dbGraph *DBGraph, config Config) ([]Delta, error) {
+func Merge(memGraph *lpg.Graph, dbGraph *DBGraph, config Config, entityActions map[string]string) ([]Delta, error) {
```

Comment:
Pass map[string]EntityMergeAction here where EntityMergeAction struct {
   Merge bool
   Create bool
}

put this in Config.